src/js/ImagePreview.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import css from '../stylesheets/button.css';
+
+export default class ImagePreview extends React.Component {
+
+   /**
+      * Find out if image src is available
+      * If not hide the image area otherwise show the image
+      * @return {String}
+   */
+   toggleImage () {
+      let image = this.props.imgSrc,
+         imgShow = classNames(css.img, css.show),
+         imgHide = classNames(css.img, css.hide);
+      return image === '' ? imgHide : imgShow;
+   }
+
+   render() {
+      // console.log(this.props);
+      return (
+         <div>
+            <img
+               src={this.props.imgSrc}
+               className={this.toggleImage()}
+            />
+            <img
```

Comment:
Is this supposed to be here? I don't see it connected to anything and the src is empty